lib/config/base.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
```

Comment:
FYI we had a `base` config before. I removed it when it didn't seem necessary https://github.com/ember-cli/eslint-plugin-ember/pull/1518. It's fine if you want to add it back. We had some other interest in adding back the `base` config here: https://github.com/ember-cli/eslint-plugin-ember/issues/1895

Reply:
yes, i think it makes sense if want to go this way and provide recommended configs per file type.
btw, eslint-typescipt also has a base config (but I didnt see it documented) https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/base.ts

Reply:
Yep, seems fine to add `base`.